code.gs

```diff
@@ -1906,6 +1906,37 @@ var i18n = {
     'Other email': 'Другой email',
     'It looks like you are using an outdated version of this script': 'Похоже вы используете устаревшую версию этой программы',
     'You can find the latest one here': 'Вы можете найти последнюю версию здесь',
+  }
```

Comment:
You are missing a comma here, after the closed curly brace.

Reply:
yeah, sorry, I missed that, fixed